recipes/libtiff/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "4.0.6" %}
+
+package:
+    name: libtiff
+    version: {{ version }}
+
+source:
+    fn: tiff-{{ version }}.tar.gz
+    url: ftp://ftp.remotesensing.org/pub/libtiff/tiff-{{ version }}.tar.gz
+    md5: d1d2e940dea0b5ad435f21f03d96dd72
+    patches:
+        - def_snprintf.patch  # [win]
```

Comment:
The closest thing I can find about needing a patch like this a [thread]( http://comments.gmane.org/gmane.comp.video.tiff/6889 ) that doesn't include the patch. I was hoping I would find exactly where this patch came from, but no such luck. I know this is tedious, sorry in advance, but would you mind submitting it upstream. You could probably even just drop it into that thread. Here is their [mailing list]( http://lists.maptools.org/mailman/listinfo/tiff ) sign up page.